### PR TITLE
Prepared Ditto 2.0.0 release notes + announcement blogpost

### PR DESCRIPTION
**Pull request by thjaeckle**

### CODE REVIEW

File: documentation/src/main/resources/pages/ditto/release_notes_200.md [hunk 1]
```diff
@@ -3,29 +3,433 @@ title: Release notes 2.0.0
...
+in a more technical way.
+
+This table shows the old modules and in which module the old ones can be found in Ditto 2.0.0:
+
+| Ditto 1.x module                             | Ditto 2.x module          |
+| ---                                          | ---                       |
+| `ditto-model`                                | `-` (was pom only)        |
+| `- ditto-model-base`                         | `ditto-base/ditto-base-model` |
+| `- ditto-model-cleanup`                      | `-` (was internal API) |
+| `- ditto-model-connectivity`                 | `ditto-connections/ditto-connections-model` |
```
Comment (Yannic92) [inline documentation/src/main/resources/pages/ditto/release_notes_200.md:115]: thew new module is named ditto-connectivity/ditto-connectivity-model

File: documentation/src/main/resources/pages/ditto/release_notes_200.md [hunk 2]
```diff
@@ -3,29 +3,433 @@ title: Release notes 2.0.0
...
+| `- ditto-model-devops`                       | `ditto-devops/ditto-devops-model` |
```
Comment (Yannic92) [inline documentation/src/main/resources/pages/ditto/release_notes_200.md:116]: ditto-model-devops was merged into ditto-base/ditto-base-api (org.eclipse.ditto.base.api.devops) and is considered to be internal API

File: documentation/src/main/resources/pages/ditto/release_notes_200.md [hunk 3]
```diff
@@ -3,29 +3,433 @@ title: Release notes 2.0.0
...
+| `- ditto-model-policies`                     | `ditto-policies/ditto-policies-model` |
+| `- ditto-model-query`                        | `ditto-rql/ditto-rql-query` |
+| `- ditto-model-rql`                          | `ditto-rql/ditto-rql-model` |
+| `- ditto-model-rql-parser`                   | `ditto-rql/ditto-rql-parser` |
+| `- ditto-model-things`                       | `ditto-things/ditto-things-model` |
+| `- ditto-model-thingsearch`                  | `ditto-thingsearch/ditto-thingsearch-model` |
+| `- ditto-model-thingsearch-parser`           | `ditto-rql/ditto-rql-parser` |
+| `ditto-protocol-adapter`                     | `ditto-protocol` |
+| `ditto-signals`                              | `-` (was pom only) |
+| `- ditto-signals-base`                       | `ditto-model-base` |
```
Comment (Yannic92) [inline documentation/src/main/resources/pages/ditto/release_notes_200.md:130]: ditto-base/ditto-base-model

File: documentation/src/main/resources/pages/ditto/release_notes_200.md [hunk 4]
```diff
@@ -3,29 +3,433 @@ title: Release notes 2.0.0
...
+| `- ditto-signals-acks`                       | `-` (was pom only) |
+| `-- ditto-signals-acks-base`                 | `ditto-base-model` |
```
Comment (Yannic92) [inline documentation/src/main/resources/pages/ditto/release_notes_200.md:132]: ditto-base/ditto-base-model

File: documentation/src/main/resources/pages/ditto/release_notes_200.md [hunk 5]
```diff
@@ -3,29 +3,433 @@ title: Release notes 2.0.0
...
+| `-- ditto-signals-acks-things`               | `ditto-things-model` |
```
Comment (Yannic92) [inline documentation/src/main/resources/pages/ditto/release_notes_200.md:133]: ditto-things/ditto-things-model

File: documentation/src/main/resources/pages/ditto/release_notes_200.md [hunk 6]
```diff
@@ -3,29 +3,433 @@ title: Release notes 2.0.0
...
+| `- ditto-signals-announcements`              | `-` (was pom only) |
+| `-- ditto-signals-announcements-base`        | `ditto-base-model` |
```
Comment (Yannic92) [inline documentation/src/main/resources/pages/ditto/release_notes_200.md:135]: ditto-base/ditto-base-model

File: documentation/src/main/resources/pages/ditto/release_notes_200.md [hunk 7]
```diff
@@ -3,29 +3,433 @@ title: Release notes 2.0.0
...
+| `-- ditto-signals-announcements-policies`    | `ditto-policies-model` |
```
Comment (Yannic92) [inline documentation/src/main/resources/pages/ditto/release_notes_200.md:136]: ditto-policies/ditto-policies-model

File: documentation/src/main/resources/pages/ditto/release_notes_200.md [hunk 8]
```diff
@@ -3,29 +3,433 @@ title: Release notes 2.0.0
...
+| `- ditto-signals-commands`                   | `-` (was pom only) |
+| `-- ditto-signals-commands-base`             | `ditto-base/ditto-base-model` |
+| `-- ditto-signals-commands-cleanup`          | `-` (was internal API) |
+| `-- ditto-signals-commands-common`           | `-` (was internal API) |
+| `-- ditto-signals-commands-connectivity`     | `ditto-connectivity/ditto-connectivity-model` |
+| `-- ditto-signals-commands-devops`           | `ditto-devops/ditto-devops-model` |
```
Comment (Yannic92) [inline documentation/src/main/resources/pages/ditto/release_notes_200.md:142]: ditto-model-devops was merged into ditto-base/ditto-base-api (org.eclipse.ditto.base.api.devops) and is considered to be internal API

File: documentation/src/main/resources/pages/ditto/release_notes_200.md [hunk 9]
```diff
@@ -3,29 +3,433 @@ title: Release notes 2.0.0
...
+| `-- ditto-signals-commands-messages`         | `ditto-messages/ditto-messages-model` |
+| `-- ditto-signals-commands-namespaces`       | `ditto-base/ditto-base-model` |
+| `-- ditto-signals-commands-policies`         | `ditto-policies/ditto-policies-model` |
+| `-- ditto-signals-commands-things`           | `ditto-things/ditto-things-model` |
+| `-- ditto-signals-commands-thingsearch`      | `ditto-thingsearch/ditto-thingsearch-model` |
+| `- ditto-signals-events`                     | `-` (was pom only) |
+| `-- ditto-signals-events-base`               | `ditto-base-model` |
```
Comment (Yannic92) [inline documentation/src/main/resources/pages/ditto/release_notes_200.md:149]: ditto-base/ditto-base-model

File: documentation/src/main/resources/pages/ditto/release_notes_200.md [hunk 10]
```diff
@@ -3,29 +3,433 @@ title: Release notes 2.0.0
...
+
+When updating from Ditto 1.x Java APIs (e.g. also when using the [Ditto Java client](#ditto-java-client)), the following
+packages were renamed:
+
+| Ditto 1.x package                                   | Ditto 2.x package      |
+| ---                                                 | ---                    |
+| `org.eclipse.ditto.model.base`                      | `org.eclipse.ditto.base.model` |
+| `org.eclipse.ditto.model.cleanup`                   | `-` (was internal API) |
+| `org.eclipse.ditto.model.connectivity`              | `org.eclipse.ditto.connectivity.model` |
+| `org.eclipse.ditto.model.devops`                    | `org.eclipse.ditto.devops.model` |
```
Comment (Yannic92) [inline documentation/src/main/resources/pages/ditto/release_notes_200.md:165]: ditto-model-devops was merged into ditto-base/ditto-base-api (org.eclipse.ditto.base.api.devops) and is considered to be internal API

**Review by Yannic92 — CHANGES_REQUESTED** [review]

**Comment by stmaute** [comment]
Fixed requested changes.

**Review by Yannic92 — APPROVED** [review]

**Pull request merged**